docs/source/reference/config.rst

```diff
@@ -484,6 +484,16 @@ Available fields and semantics:
     # Default: 'SERVICE_ACCOUNT'.
     remote_identity: my-k8s-service-account
 
+    # Allowed context names to use for Kubernetes clusters (optional).
+    #
+    # If not specified, only the current active context is used for launching.
+    #
+    # When specified, SkyPilot will fail over through the contexts in the same
+    # order as they are specified here.
```

Comment:
Some rewording and example:
```suggestion
    # SkyPilot will try provisioning and fail over Kubernetes contexts in the same order 
    # as they are specified here. E.g., SkyPilot will try using context1 first. 
    # If it is out of resources or unreachable, it will fail over and try context2.
    #
    # If not specified, only the current active context is used for launching new clusters.
```